### PR TITLE
docs(media-store): Updates for 3.x release. Removing vestigial module.

### PR DESCRIPTION
**Pull request by cjpillsbury**

### CODE REVIEW

File: docs/src/pages/docs/en/migration/from-2.x-to-3.0.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+title: Migrating from v1.x to v2.0
```
Comment (luwes) [inline docs/src/pages/docs/en/migration/from-2.x-to-3.0.md:2]: from v2...

File: docs/src/pages/docs/en/migration/from-2.x-to-3.0.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      },
+      { once: true }
+    );
+  </script>
+```
+
+**After**
+
+```html
+  <div class=".some-ancestor-element">
```
Comment (luwes) [inline docs/src/pages/docs/en/migration/from-2.x-to-3.0.md:53]: dot

File: docs/src/pages/docs/en/migration/from-2.x-to-3.0.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+### Example of media state change event usage change
+
+If you were relying on Media Chrome's media state change events (events that Media Controller dispatched whenever the corresponding media state changed; [example](https://media-chrome.mux.dev/examples/vanilla/state-change-events-demo.html)), you'll now want to add your event listeners directly to the Media Controller instance. That's it.
+**_NOTE:_** This should not impact events propagating across web component boundaries, such as themes or bundled web component players like [Mux Player](https://www.mux.com/player), as these state change events are still [composed](https://developer.mozilla.org/en-US/docs/Web/API/Event/composed).
+
+**Before**
+
+```html
+  <div class=".some-ancestor-element">
```
Comment (luwes) [inline docs/src/pages/docs/en/migration/from-2.x-to-3.0.md:27]: the dot

Comment (cjpillsbury) [reply docs/src/pages/docs/en/migration/from-2.x-to-3.0.md:27]: lol whoooops

**Review by luwes — APPROVED** [review]
🚀

**Pull request merged**